scripts/rust_lint.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+
+# This assumes you have already installed cargo-sort:
+# cargo install cargo-sort
+#
+# The best way to do this however is to run scripts/dev_setup.sh
+
+# Make sure we're in the root of the repo.
+cd "$(dirname "$0")"
```

Comment:
nit: usually I'd rather assert that we're at the root, rather than cd to it

Reply:
Open to changing this if you feel strongly about it / others agree, idm.